packages/integrations/deno/src/server.ts

```diff
@@ -1,4 +1,9 @@
+// NOTE(fks): Side-effect -- shim.js must run first. This isn't guaranteed by
+// the language, but it is a Node.js behavior that we rely on here. Keep this
```

Comment:
I don't think this is true, I think that it is guaranteed by the language. Imports are loaded asynchronously but *executed* sync after they have been fetched/parsed/etc.

Reply:
Oh nice! It looks like this may have been changed ~2020 since I last looked into it, see this answer: https://stackoverflow.com/a/35552247

> What is stopping `organize-imports-cli` from reordering this to not be first?

Because this uses TypeScript to organize the imports, we can rely on this behavior as a documented feature of TS as of 4.7. This was what had blocked this previously: https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#group-aware-organize-imports

Reply:
Ok, so what's preventing this from being reordered is because it's grouping by comments, so anything outside of this comment cannot be reordered above it, right? If so that makes sense.

Reply:
Yes that's my understanding!

Reply:
Pretty much! The comments themselves aren't even necessary, just some whitespace is needed. The blog post just seems to use comments to illustrate the problem that TS "Organize Imports" couldn't handle comments in imports. 

For example, this would also format as 2 distinct groups vs. 1 big group of 6 imports:

```
import * as aaa from "./aaa";
import * as ccc from "./ccc";
import * as bbb from "./bbb";

import * as path from "path";
import * as fs from "fs";
import * as child_process from "child_process";
```

tbh I thought that that was existing behavior!